server/src/test/java/com/linecorp/line/auth/fido/fido2/server/helper/MdsV3MetadataHelperTest.java

```diff
@@ -0,0 +1,91 @@
+package com.linecorp.line.auth.fido.fido2.server.helper;
+
+import com.linecorp.line.auth.fido.fido2.server.config.MdsConfig;
+import com.linecorp.line.auth.fido.fido2.server.config.MdsInfo;
+import com.linecorp.line.auth.fido.fido2.server.entity.MetadataEntity;
+import com.linecorp.line.auth.fido.fido2.server.exception.MdsV3MetadataException;
+import com.linecorp.line.auth.fido.fido2.server.repository.MetadataRepository;
+import org.apache.commons.io.IOUtils;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.annotation.Rollback;
+
+import javax.transaction.Transactional;
+import java.io.File;
+import java.nio.charset.StandardCharsets;
+import java.security.cert.CertificateException;
+import java.util.Collections;
+import java.util.List;
+
+@Transactional
+@Rollback
+@SpringBootTest
+class MdsV3MetadataHelperTest {
+
+    public static final int EXPECTED_METADATA_SIZE = 110;
```

Comment:
Because of U2F certificate being incorrectly accepted as FIDO2 certificate, the actual count is 73.